waiter/src/waiter/core.clj

```diff
@@ -454,6 +455,9 @@
    :http-client (pc/fnk [[:settings [:instance-request-properties connection-timeout-ms]]]
                   (http-client-factory connection-timeout-ms))
    :instance-rpc-chan (pc/fnk [] (async/chan 1024)) ; TODO move to service-chan-maintainer
+   :last-request-time-agent (pc/fnk [] (agent {:last-published {:service-id->last-request-time {}
+                                                                :time nil}
+                                               :pending {:service-id->last-request-time {}}}))
```

Comment:
Why aren't we just keeping the latest request times of all services?  Sending only diffs seems more complex.  How do we handle populating a newly restarted router's request times?